addons/l10n_th/views/report_invoice.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="report_invoice_document" inherit_id="account.report_invoice_document">
+        <xpath expr="//span[@t-field='o.partner_id.vat']" position="after">
+            <t t-if="o.company_id.account_fiscal_country_id.code == 'TH'" >
+                <span t-field="o.l10n_th_company_info" />
+                <span t-field="o.l10n_th_branch_number" />
+            </t>
+        </xpath>
+        <xpath expr="//h2" position="replace">
```

Comment:
Isn't there a better way to do this without replacing the whole h2 ?

Reply:
I think we would need to change the attribute of every line inside h2 anyway, seems like replacing it altogether is the more efficient way

Reply:
Can't you add your h2 before with a t-if use_custom_l10n_header
And add a t-else on the existing h2